website/sidebars.json

```diff
@@ -169,8 +169,8 @@
       "development/extensions-core/simple-client-sslcontext",
       "development/extensions-core/stats",
       "development/extensions-core/test-stats",
+      "development/extensions-core/azure",
```

Comment:
super nit that doesn't matter at all: maybe put up by avro so its in alphabetical order

Reply:
fixed